### PR TITLE
Crate impl

### PR DESCRIPTION
**Pull request by hwaastad**
I've not been able to verify if the change in jdbc module would have an negative effect on other jdbc sources, but I doubt that.

### CODE REVIEW

**Comment by busbey** [comment]
please update commit messages

* on the commit(s) that impact the general JDBC client please preface initial message with "[jdbc]"
* on the commits that are just for crate, please preface with "[crate]"

File: crate/Readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+# Crate Driver for YCSB
+This driver is a binding for the YCSB facilities to operate against a Crate Server cluster. It uses the official Crate JDBC and provides a rich set of configuration options.
+
+# Running tests
+
+Preferrable is to run using docker
```
Comment (busbey) [inline crate/Readme.md]: Is this the recommended way to e.g. run crate in production? Maybe better to phrase as "To get started quickly, you can test out the binding against crate running in a single docker node:"

File: crate/pom.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+Copyright (c) 2012 - 2015 YCSB contributors. All rights reserved.
```
Comment (busbey) [inline crate/pom.xml]: 2012-2014?

File: pom.xml [hunk 3]
```diff
@@ -118,6 +118,7 @@ LICENSE file.
     <module>redis</module>
     <module>tarantool</module>
     <!--<module>voldemort</module>-->
+    <module>crate</module>
```
Comment (busbey) [inline pom.xml]: please maintain lex sorting of datastore binding modules

**Comment by busbey** [comment]
someone should, at a minimum, confirm the updated jdbc driver against postgres and mysql.

**Comment by hwaastad** [comment]
Hi,
I'll fix issues an target another pull request when done.

regards,
hw

File: crate/pom.xml (hunk 2 above)
Comment (hwaastad) [inline crate/pom.xml]: I just copied header from the other module.
please inform if not applicable

Comment (allanbank) [inline crate/pom.xml]: Since this is a new file it should just be 2015.

**Comment by busbey** [comment]
when this lands, I think we can close #128

File: bin/ycsb [hunk 4]
```diff
@@ -69,7 +69,8 @@ DATABASES = {
     "orientdb"     : "com.yahoo.ycsb.db.OrientDBClient",
     "redis"        : "com.yahoo.ycsb.db.RedisClient",
     "tarantool"    : "com.yahoo.ycsb.db.TarantoolClient",
-    "voldemort"    : "com.yahoo.ycsb.db.VoldemortClient"
+    "voldemort"    : "com.yahoo.ycsb.db.VoldemortClient",
+    "crate"        : "com.yahoo.ycsb.db.JdbcDBClient"
```
Comment (busbey) [inline bin/ycsb]: please keep this is alphabetic order

**Comment by busbey** [comment]
@hwaastad are you still interested in finishing this up? there's a few nits and we need a rebase for hte current master branch.

**Comment by hwaastad** [comment]
I'll remove this one since using straight jdbc is the way to go.

**Pull request closed without merging**